pkgs/applications/editors/zee/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub, rustPlatform, openssl, pkg-config, Security }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "zee";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "zee-editor";
+    repo = pname;
+    rev = "107347474c6e63a6be6e8b1f4415f3270a92b2dc";
```

tag name is always preferred when it's available
```suggestion
    rev = "v${version}";
```